### ISSUE TITLE
connectbot resumes incorrect session when multitasking

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dragonlinux/connectbot#286]
```
What steps will reproduce the problem?
1. Start connectbot and connect to host A
2. Press back and then connect to host B
3. Switch to another task and long press home to go back to connectbot
4. You are now on host A
5. Exit from host A and select yes to close session
6. Repeat step 3

What is the expected output? What do you see instead?

After step 3 and 6 I expect to be on host B.  Step 6 is even worse because 
connectbot 
reconnects to host A!!

What version of the product are you using (you can see this by using Menu
-> About in the Host List)?

1.6.2 r480

What type of system are you trying to connect to?

Unix

```

Original issue reported on code.google.com by `pfngu...@gmail.com` on 15 Mar 2010 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Hmm, I seem to have been a little confused.  The problem is really step 6 and 
exiting from host B 
beforehand.  Upon resume of connectbot, a new connection to host B is 
established even though the 
session has already been exited and closed.
```

Original comment by `pfngu...@gmail.com` on 17 Mar 2010 at 8:50
